core/src/operator.rs

```diff
@@ -641,11 +641,12 @@ where
         }
 
         if winternitz_pubkeys.len() != self.config.get_num_kickoff_winternitz_pks() {
-            return Err(BridgeError::Error(format!(
+            return Err(eyre::eyre!(format!(
```

Comment:
no need for format! within eyre::eyre,

you can do a global replace and remove the format!